repos/fdbt-site/cypress_tests/cypress/support/helpers.ts

```diff
@@ -329,7 +329,7 @@ export const randomlyDeterminePurchaseType = (isOtherProduct?: boolean): void =>
                     if (isOtherProduct) {
                         purchaseType = $radio.attr('value');
                         purchaseType = JSON.parse(purchaseType).name;
-                        cy.get(`[id=price-${randomNumber}]`).then(($radio) => {
+                        cy.get(`[id$=price-${randomNumber}]`).then(($radio) => {
```

Comment:
getElementById ?

Reply:
can't as it looks for  id ending with price-